crates/astria-sequencer/local.env.example

```diff
@@ -33,6 +33,10 @@ ASTRIA_SEQUENCER_METRICS_HTTP_LISTENER_ADDR="127.0.0.1:9000"
 # `ASTRIA_SEQUENCER_FORCE_STDOUT` is set to `true`.
 ASTRIA_SEQUENCER_PRETTY_PRINT=false
 
+# If true disables streaming optimistic blocks to clients.
```

Comment:
I think I flagged this before and you closed the issue with a thumbsup:

```
# Disables streaming optimistic blocks to clients
```

Reply:
my bad. i forgot to make this change. will do it.